src/pycountry/db.py

```diff
@@ -30,6 +30,11 @@ def __repr__(self):
     def __dir__(self):
         return dir(self.__class__) + list(self._fields)
 
+    def __iter__(self):
```

Comment:
Enables `dict(obj)`. This can be really useful in an application, e.g. if country information is part of a JSON payload, this makes the conversion straightforward.